### PR TITLE
Fixing flaky tests on save multiple posts

### PR DESCRIPTION
**Pull request by jespino**
#### Summary
Fixing flaky tests on save multiple posts

#### Ticket Link
[MM-23226](https://mattermost.atlassian.net/browse/MM-23226)

### CODE REVIEW

**Comment by jespino** [comment]
@agnivade if the 2 milliseconds doesn't fix the problem, the underneath problem can be addressed, but this is the quick solution, and makes sense, because is checking the time there.

File: store/storetest/post_store.go [hunk 1]
```diff
@@ -94,6 +94,7 @@ func testPostStoreSave(t *testing.T, ss store.Store) {
 		replyPost.Message = "zz" + model.NewId() + "b"
 		replyPost.RootId = rootPost.Id
 
+		time.Sleep(2 * time.Millisecond)
```
Comment (hanzei) [inline store/storetest/post_store.go]: Would a delay of one millisecond be enough?

Comment (hanzei) [inline store/storetest/post_store.go]: Leaving a comment with a link to the JIRA would help other people understand why this `sleep` was needed is not a hack.

Comment (jespino) [reply store/storetest/post_store.go]: Yes, should be enough

**Comment by jespino** [comment]
@hanzei done

**Pull request merged**